app/controllers/lessons_controller.rb

```diff
@@ -89,13 +89,18 @@ def update
     redirect_to lesson_url(@lesson.id)
   end
 
+  def publish
+    @lesson = Lesson.find(params[:id])
+    @lesson.publish
+ 
+    flash["info"] = "This lesson has been published to Twitter."
+    redirect_to lesson_url(@lesson.id)
+  end
+
   def destroy
```

Comment:
@lesson is defined by the find_lesson method that gets run before_action up at the top. I can switch to the destroy.